.circleci/config.yml

```diff
@@ -53,6 +53,13 @@ jobs:
           command: |
             kubectl cluster-info
             kubectl get pods --all-namespaces
+      - run:
+          name: Update pyenv
+          command: |
+            cd /opt/circleci/.pyenv/plugins/python-build/../.. 
+            git pull
+            cd -
+      - run: pyenv install -s
```

Comment:
What do you think of expanding this argument to it's full form? `--skip-existing`. Took a moment to look up what this `-s` means.

Reply:
```suggestion
      - run: pyenv install --skip-existing
```